app/views/includes/head.jade

```diff
@@ -24,33 +24,11 @@ head
 
   link(rel='stylesheet', href='/lib/bootstrap/dist/css/bootstrap.css')
   link(rel='stylesheet', href='/css/landing.css')
-<<<<<<< HEAD
   link(rel='stylesheet', href='/css/chat.css')
   link(rel='stylesheet', href='/css/modal.css')
   link(rel='stylesheet', href='/css/modal.css')
-=======
```

Comment:
I am curious to know who did this. Please hit me via Dm if you did. We all need to be very conscious and pay attention to details whilst rebasing, so as not to break approved code in the process.
@demiladebamgbose @andela-abisoye @bolorundurowb @andela-oolutola @aezumezu @cyrielo 👆